src/python/pants/backend/python/goals/export.py

```diff
@@ -109,7 +113,20 @@ async def export_virtualenv(
     return ExportResult(
         f"virtualenv for the resolve '{request.resolve}' (using {min_interpreter})",
         dest,
-        symlinks=[Symlink(venv_abspath, py_version)],
+        digest=pex.digest,
+        post_processing_cmds=[
+            PostProcessingCommand(
+                [
+                    "{digest_root}/requirements.pex/__main__.py",
+                    "venv",
+                    "--pip",
+                    "--collisions-ok",
+                    f"{{digest_root}}/{py_version}",
+                ],
+                {"PEX_TOOLS": "1"},
+            ),
+            PostProcessingCommand(["rm", "-rf", "{digest_root}/requirements.pex"]),
```

Comment:
The `venv` tool can clean up for you, then you don't have to be bothered by the nagging concern `rm` may not exist / be on the PATH:
```
  --rm {pex,all}, --remove {pex,all}
                        Remove the PEX after creating a venv from it if the 'pex' value is specified; otherwise, remove the PEX and the PEX_ROOT if the 'all' value is specified. (default: None)
```

Reply:
Ah, but if you use the Pex PEX trick to skip re-building the requirements.pex the Pex PEX won't be removed and you'll still need a manual `rm` step :/

Reply:
I guess it doesn't have to be a VenvPex but can be a regular pex.

Reply:
Ah right, the digests are for extra Pants cruft surrounding the PEX file that Pex knows nothing of. Yeah, no need to warp the native format. If its VenvPex and you're gunning for perf, you want to leave it VenvPex and resort to `rm ...` to cleanup all but the venv dir I'd think.

Reply:
This all works if I request a regular `Pex` from the `RequirementsPexRequest`, and AFAICT that is what we usually request (e.g., for pytest runner), so I see no need to go via VenvPex. I think the only reason it was that way originally was to symlink directly to the venv pex, which we're no longer doing.

Reply:
Gotcha, sg.